tests/unit/test_snap.py

```diff
@@ -543,6 +543,27 @@ def test_cohort(self, mock_subprocess):
             universal_newlines=True,
         )
 
+    @patch("charms.operator_libs_linux.v2.snap.subprocess")
+    def test_revision_doesnt_refresh(self, mock_subprocess):
+        mock_subprocess.check_output = MagicMock()
```

Comment:
Would it be simpler just to mock `subprocess.check_output` directly? Then it would automatically be a `MagickMock`, right?